File: app/src/main/java/com/neome/api/meta/base/dto/EntVdExecuteSubWorkflow.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdPipelineParam;
import com.neome.api.meta.base.Types.MetaIdPipelineVar;
import com.neome.api.meta.base.Types.MetaIdVdAutoNode;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class EntVdExecuteSubWorkflow extends EntVdAutoStepWithOutputAndError
{
  @Nullable
  public Map<MetaIdPipelineVar, MetaIdPipelineParam> sharedParamMap;

  @Nullable
  public MetaIdVdAutoNode startNodeId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdExecuteSubWorkflow.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStepWithOutputAndError
import com.neome.api.meta.base.Types.MetaIdPipelineParam
import com.neome.api.meta.base.Types.MetaIdPipelineVar
import com.neome.api.meta.base.Types.MetaIdVdAutoNode

interface EntVdExecuteSubWorkflow : EntVdAutoStepWithOutputAndError
{
  val sharedParamMap: Map<MetaIdPipelineVar, MetaIdPipelineParam>?
  val startNodeId: MetaIdVdAutoNode?
}